### DISCUSSION TITLE
Feature Request: Horizontal buttons stack: all element types as child elements and not only buttons

### DISCUSSION THREAD

**Discussion by lexusburn** [Clooos/Bubble-Card#313] [Feature requests]
It would be great to create different entity types as child elements instead of these 1_* attributes.

Other extension had this implemented and gives us much more possibilities like conditionals and some more.
It's name is "custom:mushroom-chips-card"

A mix-up would be great: your possibility to have a fixed footer with scrolling and dynamic ordering and his possibility to use various entity types. :-)

**Comment by Clooos** [comment]
Hi and thanks for your idea! 

This could be interesting indeed, I could add the same approach that I use with pop-ups to convert an horizontal stack into a scrollable footer.

I kinda love this! Thanks again and just wait for it now 😃

**Comment by marmoul70** [comment]
A great idea to want to use the mushroom cards in the horizontal menu. Looking forward to the next update. Thank you so much

**Comment by rcjkierkels** [comment]
I would like to put chips or cards in the horizontal button stack (footer) instead of links to popups. So that I can show in the footer a warning of doors and windows that are open. When no door is open the footer can be empty.

**Comment by SLaks** [comment]
Simpler version of this FR (should I file separately?):

Allow sub-buttons inside the buttons in a horizontal stack.

Example uses:

 - A button that opens a popup with every light in the second floor, with a sub button to turn them all off without opening the popup
 - A button that opens https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card, with a sub button or 2 to immediately run the vacuum.

Relatedly, I'd love to have separators or spacers between buttons in the horizontal stack (I can probably fake that with `card_mod`)
